data/zilliqa/assets/zil1qzwxvp3wz3q5ewzg3afesut9pqfax0sjdyna42/meta.json

```diff
@@ -1,9 +1,16 @@
 {
   "name": "Messi Fan Token",
   "symbol": "MESSI",
+  "web": "http://messitokens.me",
   "supply": "100000",
+  "email": "info@messitokens.me",
+  "publicTeam": true,
```

Comment:
remove, no team listed on linkedin. thumbnails with a brief bio and non-fullnames on the sites don't count as it can easily be faked.

Reply:
Ok, let me put full names and link to LinkedIn page for each of us.

Reply:
Balthazar, the thing is this. We all have jobs (at this moment) if we put that we are employees of Messi Tokens (we are just starting) we could get in trouble. We are going to put our full name and the linkedin URL to our profiles.  Can we give you any proof of validity writing something in our profiles?

Reply:
Sure, I'll ask for one or two connections with the profiles to check

Reply:
Awesome. We are updating the site right now. We upgraded to https. I will let you know once we have our names complete with the linkedin url for each of us. Thank you :)

Reply:
Done! all the profiles now have full name and the LinkedIn button for you to go. We all have linkedIn except Angel. I hope that is not an issue.

![image](https://user-images.githubusercontent.com/36340953/121831384-b4316580-cc9d-11eb-9520-8bca026b7f39.png)